### ISSUE TITLE
`impt dg create` doesn't support `--load-code-after-blessing`?

### ISSUE THREAD

**Issue by rlipscombe** [electricimp/imp-central-impt#28]
At least the help text suggests it's not supported.

**Comment by ppetrosh** [comment]
It doesn't prevent the tool from making this request for the user after the DG is created...

**Comment by alexey-nbl** [comment]
Yes, can be supported by impt using two requests. But I would not do it as:
- there seems no real benefit. Assuming a script can easily do the same using two impt commands.
- and a downside is one more complexity in already huge impt.

But if it is really required, may be added into impt.

**Comment by ppetrosh** [comment]
I don't have a strong opinion here and would let @rlipscombe comment.

**Comment by rlipscombe** [comment]
I could live with some extra help text in `impt dg create` that said something along the lines of "use impt dg update for other options".
